public/regdesk.html

```diff
@@ -199,7 +209,77 @@ <h4>Payment required</h4>
       </div>
     </div>
   </div>
-  <div id="manualPrintModal" class="modal fade" tabindex="-1" aria-labelledby="manualPrintModalTitle" aria-hidden="true">
+  <div id="printerOptionModal" class="modal fade" tabindex="-1">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <form id="printerSettingsForm">
+          <div class="modal-header">
+            <h5 class="modal-title">Printer Settings For: <span id="printerSettingsSerial"></span></h5>
+            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+          </div>
+          <div class="modal-body">
+            <div class="row">
+              <div class="col-md-12 alert alert-danger">
+                Do not change the configuration when the label dispenser is engaged. It will dispense many labels and
+                not stop. Ensure dispenser is disengaged before saving.
```

Comment:
What does this mean? There isn't a button called "disengage". Is a human supposed to interpret that to mean "while printing"? Perhaps it would be simpler to say "while printing".

Reply:
This is the boundary where we expect someone to know what they're doing when reading this. This page here will feature in the help documentation I've started writing that will be in a physical folder with the printers.

![image](https://user-images.githubusercontent.com/1441553/148002150-6b3cab6c-6a5a-433f-95ea-d6df66851f7c.png)